### ISSUE TITLE
Further calibrate model 

### ISSUE THREAD

**Issue by huevosabio** [huevosabio/ts-predict#9]
That is, continue to "center" the histograms! 

Some hints: 
- **Use Mean Absolute Error (MAE)**: Mean Square Error (MSE) basically tends to estimate your _average_, while MAE tends to estimate the _median_, this can help us. Additionally, it is theorically possible to get even more savvy for heavy tailed distributions (think distributions with big outliers), some background at: http://www.mitpressjournals.org/doi/pdf/10.1162/08997660260293300
- **Use Concrete Dropout**, as it turns out, part of the issue is that the dropout probabilities are parameters themselves (remember the hyperparameter and grid search thing we talked about?). However, our friend Yarin Gal who came up with the uncertainty things, also came up with a way to estimate the desired dropout probabilities, the method is called ConcreteDropour. You can read more here https://arxiv.org/abs/1705.07832, check the code at the very end, it is in Keras!

**Comment by huevosabio** [comment]
Also, in the DistributionTests notebook, I added a section at the end comparing a LogNormal and Normal histograms as we would normally do in the predictions. Note that **I was wrong** we don't want the histogram to do a "nice" bell curve over the CDFs, what we want it to be _uniformly distributed_. Ping me if you have questions

**Comment by huevosabio** [comment]
@tomiyee, On the distribution stuff:
-https://forecasters.org/wp-content/uploads/gravity_forms/7-621289a708af3e7af65a7cd487aee6eb/2015/07/Kolassa_Stephan_ISF2015.pdf
